keel-api-jackson/src/test/kotlin/com/netflix/spinnaker/keel/api/DeliveryArtifactTests.kt

```diff
@@ -83,4 +84,10 @@ private data class Fixture(
   val json: String
 ) {
   val mapper = configuredObjectMapper()
+    .also {
+      it.registerSubtypes(
```

Comment:
A small annoyance resulting from this change is that, because support for docker and debian is now provided by plugins, you have to register the artifact sub-types when you need them.

Reply:
oof. Any way we could do this automatically in the `keel-test` module or something? Maybe that's a very ignorant question. But feels like we could have a `testConfiguredObjectMapper()` method that does that?

Reply:
Ah, no, that's a good idea. Let me work something out!